sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-SBSD.e2e-spec.ts

```diff
@@ -0,0 +1,73 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../test-utils";
+import { YesNoOptions } from "@sims/test-utils";
+import { InstitutionTypes } from "../../models";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-SBSD.`, () => {
+  it(
+    `Should determine SBSD as eligible when institution type is ${InstitutionTypes.BCPrivate}, ` +
+      "total assessment need is greater than 0 and application PD/PPD status is 'yes' for single student.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.institutionType = InstitutionTypes.BCPrivate;
```

Comment:
A similar approach done for full-time could be used here.
[sources\packages\backend\workflow\test\2023-2024\fulltime-assessment\eligibility\fulltime-assessment-2023-2024-eligibility-BGPD.e2e-spec.ts]https://github.com/bcgov/SIMS/blob/4977cf78ea5573182043c7b47634ca6cc861b4f0/sources/packages/backend/workflow/test/2023-2024/fulltime-assessment/eligibility/fulltime-assessment-2023-2024-eligibility-BGPD.e2e-spec.ts#L10-L16

Reply:
Same from https://github.com/bcgov/SIMS/pull/2506#discussion_r1394997176